src/sentry/static/sentry/app/components/events/interfaces/threads/trim-filename.ts

```diff
@@ -0,0 +1,6 @@
+function trimFilename(filename: string) {
```

Comment:
I think we are using .tsx extension everywhere, even if the file does not contain jsx.